### PR TITLE
Updated link on READ.ME file + added Application Form link(to Apply to be a MahaDev)

### PR DESCRIPTION
**Pull request by NidhxGupta**
I have updated the link on READ.ME and _config.yml file as was mentioned.

I have also added a place where the Application Form link can be added for applying to be a MahaDev. 

I do have a question, Is the application form ready? Or will we have to create one?
Because if it already exists, we can just add the link in that place.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -9,6 +9,6 @@ description: > # this means to ignore newlines until "baseurl:"
   We are makers. We are hackers. We build things. We break things. We code, we learn, we code, we play, we teach, we code, we sleep, we code, we think, we code.<br><br>
   And we want you to join us!
 # baseurl: "/" # the subpath of your site, e.g. /blog/
-url: "http://devs.mozillaindia.org" # the base hostname & protocol for your site
+url: "http://mozillaindia.org/devs/" # the base hostname & protocol for your site
```
Comment (asdofindia) [inline _config.yml]: The /devs/ part will have to be in the baseurl (the line above)

File: start.md [hunk 2]
```diff
@@ -54,4 +54,4 @@ The things you will definitely need are:
 
 ## Apply to be a MahaDev ##
 
-If you haven't done so already, fill this application form and one of us will get in touch with you.
+If you haven't done so already, fill this [application form](#ProvideLinkHere) and we will get in touch with you.
```
Comment (asdofindia) [inline start.md]: There is no point in this either. I think you should follow the conversation in #6

**Review by asdofindia — CHANGES_REQUESTED** [review]
Let us discuss this at #6

**Comment by NidhxGupta** [comment]
I have made the changes as requested in #10 @asdofindia

**Comment by asdofindia** [comment]
Merged via https://github.com/MozillaIndia/devs/commit/f36b0bb51014afb019320050b1183b526639334a3
Thanks

**Pull request closed without merging**